addons/xeh/fnc_compileEventHandlers.sqf

```diff
@@ -151,21 +147,17 @@ private _resultNames = [];
                 };
 
                 // client only events
-                if (!isDedicated) then {
-                    _entry = _x >> format ["client%1", _entryName];
+                _entry = _x >> "clientInit";
```

Comment:
format ["client%1", _entryName];

Reply:
I don't even know how I managed to do that

Reply:
Ah I just copied the stuff from the non-object eventhandlers. They don't have that stuff apparently